scripts/travis/upload-to-docker.sh

```diff
@@ -33,7 +33,7 @@ fi
 
 # Do not enable echo before the `docker login` command to avoid revealing the password.
 set -x
-docker login -u $DOCKER_USER -p $DOCKER_PASS
+docker login docker.io -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
docker.io redirects to docker.com shouldn't the com domain be used?

Reply:
The image is tagged for `docker.io`. When pushing `docker.io/jaegertracing/....`, the credentials for `docker.io` are used.